controllers/nc_news_controllers.js

```diff
@@ -15,7 +16,17 @@ const getTopics = (req, res, next) => {
     })
 }
 
+const getArticleByID = (req, res, next) => {
+    const { article_id } = req.params
+    selectArticleByID( article_id ).then((article) => {
+        res.status(200).send({ article: article })
```

Comment:
This line can be refactored to as follows using modern syntax that will auto assign a key with the same name

`   res.status(200).send({ article }) `